Cargo.toml

```diff
@@ -32,22 +32,21 @@ name = "store"
 harness = false
 
 [features]
-arch-arm64-sve = ["dep:cc"]
-default = ["blake3/default", "std", "winter_crypto/default", "winter_math/default", "winter_utils/default"]
+default = ["std"]
 executable = ["dep:clap", "dep:rand_utils", "std"]
-std = ["blake3/std", "dep:cc", "dep:libc", "dep:rand", "winter_crypto/std", "winter_math/std", "winter_utils/std"]
-serde = ["winter_math/serde", "dep:serde", "serde/alloc"]
+serde = ["dep:serde", "serde?/alloc", "winter_math/serde"]
+std = ["blake3/std", "dep:cc", "dep:libc", "winter_crypto/std", "winter_math/std", "winter_utils/std"]
+sve = ["std"]
```

Comment:
this is introducing a lot new features requirements to the `sve` feature, are those necessary?

Reply:
Currently, when we enable `sve` this is done in the context of building a full prover (primarily for Graviton3 platform right now) - so, brining in `std` is fine. I tried to make it more relaxed by ran into some issues which I decided to postpone til later. So, basically, once we start using `sve` in other contexts, we can come back and revisit this.